code/Sorting algorithms/insertionSort.py

```diff
@@ -0,0 +1,28 @@
+import time
+import numpy as np
+
+def swapTwoAr(arr, pos1, pos2): #not utilising built ins
+    temp = arr[pos1]
+    arr[pos1] = arr[pos2]
+    arr[pos2] = temp
+    return arr
+
+def insertionSort(A):
+    start = time.time()
+    try:
+        for ii in range(1, len(A)):
+            i = ii
+            while(i > 0) and (A[i-1] > A[i]):
+                swapTwoAr(A, i, i-1)
+                i = i - 1
+    except TypeError:
+        print("Error in input")
+    end = time.time()
+    timeT = end - start
+    duration = round(timeT, 5)
+    print("---Finished in ", duration, "seconds---")
+
+
+testArr = np.random.randint(0, 999, size=(100))
```

Comment:
Perhaps consider using list comprehension + random.randint here to generate the random numbers